Broadcom/siteminder/tests/AuthAttempt.json

```
@@ -0,0 +1,64 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Broadcom Siteminder",
+        "dialect_uuid": "6a740c4b-468b-45b4-9982-3903abf9fc54"
+      }
+    },
+    "message": "AuthAttempt HOST_XXX [23/Jul/2025:07:01:48 +0200] \"10.33.110.2 T0316290@xxxxgroup.com\" \"domain.com GET /affwebservices/redirectjsp/redirect_unique.jsp?SAMLRequest=nZJfS8MwFMXf%2FRQl72uzrl23sA7GhlBQmU588C3NblkgTWpuotu3N61%2FmAgTfEtubs65v5MskLeqYyvvDvoBXjygi6JjqzSy4aQk3mpmOEpkmreAzAm2W93esDSmrLPGGWEUuao2JcG0ridZPs%2Fq%2BVhkKWTjWV7TIuXTYp7ls6JIm3AynQpKoiewKI0uSZAhUYXoodLouHahRNN8RItROnmkOaOUZbNnEm3CaFJzN9w6ONchSxJEM5L7LhbGdrE7cAWY8KZ5gxrBvkoRtp2vlRRJT5OGdhJdGytgAC5JwxVC77%2FliPIVvisrDAK919po9C3Y3YdepfdwLMmMLK8WvSQbRrdnkV1OjH%2FpkqU7IJ72XggvWjru4rCKpW4s%2FwRZJGcGn3YduwuS1WZrAtPpPy%2FV47fcXe7uK3I%2FaoZW5izXKEE7Eu22vf%2B950o2EmxJ%2FoIISSpl3tYWuAvpOuuBLD%2FIfrIEwOT3X1y%2BAw%3D%3D&RelayState=s2bb34594b91c42e4185b072a679458772fb9166c0&SMPORTALURL=https%3A%2F%2Fsso-idp.corp.xxxx%2Faffwebservices%2Fpublic%2Fsaml2sso\" [] [0]  [] []"
+  },
+  "expected": {
+    "message": "AuthAttempt HOST_XXX [23/Jul/2025:07:01:48 +0200] \"10.33.110.2 T0316290@xxxxgroup.com\" \"domain.com GET /affwebservices/redirectjsp/redirect_unique.jsp?SAMLRequest=nZJfS8MwFMXf%2FRQl72uzrl23sA7GhlBQmU588C3NblkgTWpuotu3N61%2FmAgTfEtubs65v5MskLeqYyvvDvoBXjygi6JjqzSy4aQk3mpmOEpkmreAzAm2W93esDSmrLPGGWEUuao2JcG0ridZPs%2Fq%2BVhkKWTjWV7TIuXTYp7ls6JIm3AynQpKoiewKI0uSZAhUYXoodLouHahRNN8RItROnmkOaOUZbNnEm3CaFJzN9w6ONchSxJEM5L7LhbGdrE7cAWY8KZ5gxrBvkoRtp2vlRRJT5OGdhJdGytgAC5JwxVC77%2FliPIVvisrDAK919po9C3Y3YdepfdwLMmMLK8WvSQbRrdnkV1OjH%2FpkqU7IJ72XggvWjru4rCKpW4s%2FwRZJGcGn3YduwuS1WZrAtPpPy%2FV47fcXe7uK3I%2FaoZW5izXKEE7Eu22vf%2B950o2EmxJ%2FoIISSpl3tYWuAvpOuuBLD%2FIfrIEwOT3X1y%2BAw%3D%3D&RelayState=s2bb34594b91c42e4185b072a679458772fb9166c0&SMPORTALURL=https%3A%2F%2Fsso-idp.corp.xxxx%2Faffwebservices%2Fpublic%2Fsaml2sso\" [] [0]  [] []",
+    "event": {
+      "action": "AuthAttempt",
+      "category": [
+        "authentication"
+      ],
+      "dataset": "smaccess",
+      "kind": "event",
+      "outcome": "unknown",
+      "reason": "Sm_Api_Reason_None"
+    },
+    "@timestamp": "2025-07-23T05:01:48Z",
+    "broadcom": {
+      "siteminder": {
+        "agent": {
+          "name": "domain.com"
+        }
+      }
+    },
+    "client": {
+      "address": "10.33.110.2",
+      "ip": "10.33.110.2",
+      "user": {
+        "name": "T0316290@xxxxgroup.com"
```

Can you split this value?
```suggestion
        "name": "T0316290",
        "domain": "xxxxgroup.com"
```